src/client/nativeCommunication.ts

```diff
@@ -62,7 +62,15 @@ function insertAds(): void {
             }
         };
 
-        window.addEventListener('resize', callback);
+
+        let timeout: number;
+        window.addEventListener('resize', () => {
+            if (timeout) {
+                window.cancelAnimationFrame(timeout);
+            }
+            timeout = window.requestAnimationFrame(callback);
+        }, false);
```

Comment:
Would you be able to give a brief explanation of how this works?

Reply:
https://gomakethings.com/debouncing-your-javascript-events/#requestanimationframe

Reply:
Under what circumstances will the `timeout` in `if (timeout)` be falsy?

Reply:
The first time when there is no previous animation frame to cancel. I've renamed it as it's not really a "timeout"